rust/forevervm-sdk/src/api/api_types.rs

```diff
@@ -61,8 +61,10 @@ impl Instruction {
 
 #[derive(Debug, Serialize, Deserialize, PartialEq, Clone)]
 pub enum ExecResultType {
-    #[serde(rename = "value")]
-    Value(Option<String>),
+    Value {
```

Comment:
we still need the serde rename here, otherwise this will change to `Value` in the json representation.

We could also just put a `#[serde(rename_all="snake_case")]` between the `derive` and `pub enum ExecResultType`.